job-dsls/jobs/prod/prod_osl_trigger_umb_message.groovy

```diff
@@ -15,7 +15,7 @@ pipelineJob("${folderPath}/osl-trigger-umb-message") {
         stringParam('MILESTONE', '', 'The release milestone, i.e. 1.27.0.CR1. This parameter is optional and in case it is not defined, the latest available milestone is used.')
         stringParam('QUARKUS_PLATFORM_VERSION', '2.13.5.Final-redhat-00002', 'The productized version of Quarkus Platform used for building Openshift Serverless Logic currect milestone.')
         stringParam('QUARKUS_VERSION', '2.13.5.Final-redhat-00003', 'The productized version of Quarkus used for building Openshift Serverless Logic currect milestone.')
-        stringParam('PNC_API_URL', 'http://orch.psi.redhat.com/pnc-rest/v2', 'PNC Rest API endpoint. See: https://docs.engineering.redhat.com/display/JP/User%27s+guide')
+        stringParam('PNC_API_URL', "\${env.ORCH_PSI_URL}/pnc-rest/v2", "PNC Rest API endpoint. See: \${env.DOCS_ENGINEERING_URL}/display/JP/User%27s+guide")
```

Comment:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.